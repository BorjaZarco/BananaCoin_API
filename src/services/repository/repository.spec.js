/* global expect, describe, it */
const repositoryService = require('./')

jest.mock('util', () => {
  const fixture = require('./fixture')
  return { promisify: jest.fn(() => () => fixture) }
})

describe('service repository', () => {
  const dirname = 'dirname'
  const dbname = 'dbname'
  const repository = repositoryService(dirname, dbname)

  it('can be build', async () => {
    expect(repository).toMatchSnapshot()
  })
})
