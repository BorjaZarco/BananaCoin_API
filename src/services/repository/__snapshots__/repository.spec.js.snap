// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`service repository can be build 1`] = `Object {}`;
